app/src/main/java/io/homeassistant/companion/android/PresenterModule.kt

```diff
@@ -35,6 +37,8 @@ class PresenterModule {
     private lateinit var mobileAppIntegrationView: MobileAppIntegrationView
     private lateinit var settingsView: SettingsView
     private lateinit var webView: WebView
+    private lateinit var ssidPreference: SsidPreference
+    private lateinit var ssidDialog: SsidDialogFragment
```

Comment:
These shouldn't be loaded with dagger.  They should be treated like any of the other preferences and created when the settings screen is loaded.